Sources/PackageModel/Manifest/TraitConfiguration.swift

```diff
@@ -0,0 +1,64 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+/// The trait configuration.
+public enum TraitConfiguration: Codable, Hashable {
+    case enableAllTraits
+    case disableAllTraits
+    case enabledTraits(Set<String>)
+    case none
```

Comment:
Should we call this `default` instead?

Reply:
Yup! Renaming it to default will also get rid of the need for the static variable as well.